python/pyspark/pandas/series.py

```diff
@@ -992,8 +993,10 @@ def map(self, arg: Union[Dict, Callable]) -> "Series":
 
         Parameters
         ----------
-        arg : function or dict
+        arg : function, dict, or pd.Series
             Mapping correspondence.
+        na_action :
+            If ‘ignore’, propagate NA values, without passing them to the mapping correspondence.
```

Comment:
```suggestion
            If `ignore`, propagate NA values, without passing them to the mapping correspondence.
```

super nits: Looks it's c-p from pandas, but I think "`" it more better.

Reply:
Good catch! Modified.